sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/XmlSensor.java

```diff
@@ -61,8 +64,14 @@ public class XmlSensor implements Sensor {
   private final FilePredicate mainFilesPredicate;
   private final SonarRuntime sonarRuntime;
   private final FileLinesContextFactory fileLinesContextFactory;
+  private final Configuration configuration;
 
   public XmlSensor(SonarRuntime sonarRuntime, FileSystem fileSystem, CheckFactory checkFactory, FileLinesContextFactory fileLinesContextFactory) {
+    this(sonarRuntime, fileSystem, checkFactory, fileLinesContextFactory, null);
+  }
+
+  public XmlSensor(SonarRuntime sonarRuntime, FileSystem fileSystem, CheckFactory checkFactory, FileLinesContextFactory fileLinesContextFactory,
+    @Nullable Configuration configuration) {
```

Comment:
Nitpick: Can we move the parameter to the same line? Or indent the parameter to start on the same column as the first parameter in the same list?